packaging/tests/tas-cmis/pom.xml

```diff
@@ -58,10 +59,30 @@
     </profiles>
 
     <dependencies>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-jackson2-provider</artifactId>
+            <version>4.7.1.Final</version>
+        </dependency>
+
+        <!-- alfresco tester settings -->
         <dependency>
             <groupId>org.alfresco.tas</groupId>
-            <artifactId>cmis</artifactId>
-            <scope>test</scope>
+            <artifactId>utility</artifactId>
+            <version>${tas.utility.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>mysql</groupId>
+                    <artifactId>mysql-connector-java</artifactId>
```

Comment:
I excluded this library since I don't think it's used and because the GPL license wasn't approved for release.  I think we use postgres almost everywhere unless we're explicitly testing another DB, so I didn't think this would be a problem.